Libraries/LibWeb/IndexedDB/Internal/Algorithms.cpp

```diff
@@ -2044,4 +2044,35 @@ GC::Ref<JS::Array> retrieve_multiple_values_from_an_index(JS::Realm& realm, GC::
     return list;
 }
 
+// https://w3c.github.io/IndexedDB/#cleanup-indexed-database-transactions
+bool cleanup_indexed_database_transactions(GC::Ref<HTML::EventLoop> event_loop)
+{
+    GC::ConservativeVector<GC::Ref<IDBTransaction>> matching_cleanup_loop(event_loop->heap());
```

Comment:
`GC::ConservativeVector` doesn't really add much to `GC::Ref` values, you can use `GC::RootVector` instead